docs/tutorials/04_vqd.ipynb

```diff
@@ -0,0 +1,351 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variational Quantum Deflation (VQD) Algorithm\n",
+    "\n",
+    "This notebook demonstrates how to use Qiskit's implementation of the [Variational Quantum Deflation (VQD)](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VQD.html) algorithm for computing higher energy states of a Hamiltonian, as introduced in this [reference paper](https://arxiv.org/abs/1805.08138)."
```

Comment:
>  use Qiskit's implementation of

As this is not part of Qiskit anymore maybe `use our implementation of` or just `use the [Var] and drop the implementation  of - it should hopefully be clear that these are tutorials for the algorithms in qiskit_algorithms so it can go without saying, no?